.gitignore

```diff
@@ -198,7 +198,7 @@ cython_debug/
 #.idea/compiler.xml
 #.idea/jarRepositories.xml
 #.idea/modules.xml
-#.idea/*.iml
+.idea/*.iml
```

Comment:
Uncommented this bc it looks like we're tracking changes in the .idea/NASS-CISS-Data-Repository.iml file, which we probably don't care about

Reply:
That makes sense! Let's also then remove the changes to the .idea/NASS-CISS-Data-Repository.iml file from this PR